site/index.css

```diff
@@ -43,28 +43,57 @@ body {
   float: right;
   color: #FFFAFB;
   font-family: "Palanquin Dark";
+  font-size: 1.1em;
   /* todo: choose alt fonts */
 }
 
 #aquapants-quote {
+  margin-top: 1.5em;
   margin-bottom: 1.5em;
 }
 
 #aquapants-quote q {
   font-family: "Crimson Text";
   /* todo: choose alt fonts */
   font-style: italic;
-  font-size: 1.2em;
 }
 
+/*Tablet size*/
+@media screen and (max-height: 768px) {
+  #aquapants-header h1 {
+    font-size: 3.5em;
+  }
+
+  #aquapants-quote {
+    margin-top: 0px;
+  }
+}
+
+/*Mobile L size*/
+@media screen and (max-width: 425px) {
```

Comment:
This is why breakpoints are tricky! Here's an iPhone version that has a width of 428 px and therefore it falls under the "tablet" category. I think to be safe let's just use `620px` as our breakpoint for mobile L width.

```suggestion
@media screen and (max-width: 620px) {
```

![image](https://github.com/aquapants/website/assets/123187978/fe7c1c93-1ebc-45cf-b7fe-9caabe59841f)

Reply:
Ah, I found the problem, I was making changes to 768px, 425px and 320px mode. I accidentally put the "max-width" to "max-height", which cause the above problem.

Reply:
https://github.com/aquapants/website/pull/9/commits/876f33eb43fd54cd2969b41f1f86437157e55a68 
Fixed